.changeset/afraid-trees-stare.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': patch
+---
+
+changeset for zod to json converter
```

Comment:
Probably want to change this a little bit to add a more descriptive text about what the change actually does and how to use it I think.